src/java.management/share/classes/javax/management/remote/JMXConnectorFactory.java

```diff
@@ -447,20 +440,12 @@ static <T> T getProvider(JMXServiceURL serviceURL,
         return instance;
     }
 
-    @SuppressWarnings("removal")
     private static ClassLoader wrap(final ClassLoader parent) {
-        return parent != null ? AccessController.doPrivileged(new PrivilegedAction<>() {
-            @Override
-            public ClassLoader run() {
-                return new ClassLoader(parent) {
-                    @Override
-                    protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundException {
-                        ReflectUtil.checkPackageAccess(name);
-                        return super.loadClass(name, resolve);
-                    }
-                };
-            }
-        }) : null;
+        return parent != null ? new ClassLoader(parent) {
+                                    protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundException
+                                        { return super.loadClass(name, resolve); }
+                                }
+                              : null;
```

Comment:
I suspect we could just return `parent` here but that might require some deeper investigation.